libraries/Microsoft.Bot.Builder.Dialogs.Debugging/Base/SemaphoreSlimExtensions.cs

```diff
@@ -17,24 +16,5 @@ public static async Task<Releaser> WithWaitAsync(this SemaphoreSlim semaphore, C
             await semaphore.WaitAsync(cancellationToken).ConfigureAwait(false);
             return new Releaser(semaphore);
         }
-
-#pragma warning disable CA1034 // Nested types should not be visible (we can't change this without breaking binary compat, consider fixing this before we go out of preview)
```

Comment:
We should be able to change this, isn't this a preview library?  Why not pull these classes out into own files and mark as internal?

Reply:
done